CHANGELOG.md

```diff
@@ -13,6 +13,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Removed
 - None.
 ### Fixed
-- None.
+- Updated project to use URLs instead of String paths. https://github.com/JamitLabs/ProjLint/pull/33
```

Comment:
Could you please update this to the format of other entries in the JamitLabs account? Like [here](https://raw.githubusercontent.com/JamitLabs/Accio/stable/CHANGELOG.md)? Thanks!

Reply:
Also please note that this does not explain, what bug(s) this fixes.

Reply:
Done

Reply:
Thanks! Will merge then, once the CI succeeds.